lib/core/Screens/chatScreen.dart

```diff
@@ -1,22 +1,28 @@
+import 'package:chatapp/UIwidgets/massageWidget.dart';
 import 'package:flutter/cupertino.dart';
 import 'package:flutter/material.dart';
 import 'package:flutter/painting.dart';
 import 'package:flutter/rendering.dart';
 import 'package:flutter/widgets.dart';
 import 'package:firebase_auth/firebase_auth.dart';
 import 'package:cloud_firestore/cloud_firestore.dart';
+final _firestore =FirebaseFirestore.instance;
+late User signInUser;  //to save the user mail that will send
```

Comment:
this is the main problem this line make the variable sinInUser be able to allowed